[2025-10-17T08:29:03.829+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-17T08:29:03.846+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag3.task_pwd scheduled__2025-10-17T08:28:00+00:00 [queued]>
[2025-10-17T08:29:03.852+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag3.task_pwd scheduled__2025-10-17T08:28:00+00:00 [queued]>
[2025-10-17T08:29:03.852+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-17T08:29:03.864+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_pwd> on 2025-10-17 08:28:00+00:00
[2025-10-17T08:29:03.870+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=631) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-17T08:29:03.872+0000] {standard_task_runner.py:63} INFO - Started process 636 to run task
[2025-10-17T08:29:03.872+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag3', 'task_pwd', 'scheduled__2025-10-17T08:28:00+00:00', '--job-id', '157', '--raw', '--subdir', 'DAGS_FOLDER/dag3.py', '--cfg-path', '/tmp/tmpqo2ub2h2']
[2025-10-17T08:29:03.874+0000] {standard_task_runner.py:91} INFO - Job 157: Subtask task_pwd
[2025-10-17T08:29:03.886+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-17T08:29:03.921+0000] {task_command.py:426} INFO - Running <TaskInstance: dag3.task_pwd scheduled__2025-10-17T08:28:00+00:00 [running]> on host 9565199fb8e8
[2025-10-17T08:29:03.992+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag3' AIRFLOW_CTX_TASK_ID='task_pwd' AIRFLOW_CTX_EXECUTION_DATE='2025-10-17T08:28:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-17T08:28:00+00:00'
[2025-10-17T08:29:03.994+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-17T08:29:03.995+0000] {logging_mixin.py:188} INFO - 1
[2025-10-17T08:29:03.995+0000] {logging_mixin.py:188} INFO - 2
[2025-10-17T08:29:03.995+0000] {logging_mixin.py:188} INFO - 3
[2025-10-17T08:29:03.996+0000] {logging_mixin.py:188} INFO - 4
[2025-10-17T08:29:03.996+0000] {logging_mixin.py:188} INFO - 5
[2025-10-17T08:29:03.996+0000] {logging_mixin.py:188} INFO - 6
[2025-10-17T08:29:03.996+0000] {logging_mixin.py:188} INFO - 7
[2025-10-17T08:29:03.997+0000] {logging_mixin.py:188} INFO - 8
[2025-10-17T08:29:03.997+0000] {logging_mixin.py:188} INFO - 9
[2025-10-17T08:29:03.997+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-17T08:29:03.998+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-17T08:29:04.005+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag3, task_id=task_pwd, run_id=scheduled__2025-10-17T08:28:00+00:00, execution_date=20251017T082800, start_date=20251017T082903, end_date=20251017T082904
[2025-10-17T08:29:04.046+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-17T08:29:04.059+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-17T08:29:04.061+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
